client/src/context.rs

```diff
@@ -0,0 +1,127 @@
+use crate::{texture, window::Window};
```

Comment:
Could you remove the context file if we're not using it?

Reply:
Thanks for catching that! I've removed it.